api/src/pcapi/alembic/versions/20240111T195604_cdb9a9f92d2b_add_partial_index_on_offer_extradata_for_music_type.py

```diff
@@ -0,0 +1,39 @@
+"""
+add partial index on offer extraData for music_type
```

Comment:
```suggestion
Add partial index for musicType in Offer.extraData
```
C'est plus joli :p 
(à faire pour les 4 scripts 😉  )